include/secrets.example.h

```diff
@@ -37,5 +37,7 @@
 #define cszOpenWeatherAPIKey ""                     // Your OpenWeatherMap API key goes Here
 #define cszLocation          "98074"
 #define bLocationIsZip       true
+#define bUse24HourClock      false
+#define bUseCelsius          false

```

Comment:
I don't think we need to, or should, add these. These settings are only used on devices/projects that have the web UI enabled, and that is the way to change the default. Coming from a different angle, there's a whole heap of other settings for which secrets(.example).h doesn't contain a define. If I'm totally honest I considered taking defines _out_ of here while I was working on the first iteration of `DeviceConfig`, but I figured grandfathering what was already there anyway was ok.

Reply:
The Celsius setting is used in weather and I promoted it to secrets so that I could change it without affecting my commit set as I have to run it without the web UI to get it to fit in my paltry 4MB board.  I can certainly revert this.